apps/site/lib/site_web/templates/schedule/_retirement_message.html.eex

```diff
@@ -0,0 +1,4 @@
+<div class="stop-filters schedule-filter-row callout">
```

Comment:
```suggestion
<div class="callout">
```

Looks like you copied this from elsewhere! Try to use the most general classes needed to achieve the desired effect, here it looks like only the `.callout` class is necessary?